### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by garrettg123** [webpro/knip#486]
```
> npx knip
Need to install the following packages:
knip@4.2.3
Ok to proceed? (y)
Analyzing source files...
file:///Users/gsquare567/.npm/_npx/8d873e0e769cf1b4/node_modules/knip/dist/IssueFixer.js:28
        if (fix.length === 0)
                ^

TypeError: Cannot read properties of undefined (reading 'length')
    at IssueFixer.addUnusedExportNode (file:///Users/gsquare567/.npm/_npx/8d873e0e769cf1b4/node_modules/knip/dist/IssueFixer.js:28:17)
    at main (file:///Users/gsquare567/.npm/_npx/8d873e0e769cf1b4/node_modules/knip/dist/index.js:478:35)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (file:///Users/gsquare567/.npm/_npx/8d873e0e769cf1b4/node_modules/knip/dist/cli.js:24:73)
    at async file:///Users/gsquare567/.npm/_npx/8d873e0e769cf1b4/node_modules/knip/dist/cli.js:79:1

Node.js v20.9.0
```

**Comment by webpro** [comment]
There's not much I can do without a reproduction of the issue.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v4.3.1. See [Release 4.3.1](https://github.com/webpro/knip/releases/tag/4.3.1) for release notes._

**Comment by webpro** [comment]
Added some safe-guards so at least Knip won't halt like this.
